packages/functionals/botpress-qna/src/providers/qnaMaker.js

```diff
@@ -0,0 +1,174 @@
+import axios from 'axios'
+import _ from 'lodash'
+import ms from 'ms'
+import Promise from 'bluebird'
+
+// Handles QnA Maker API downcasing all key-values in metadata
+const markUpperCase = str => str.replace(/([A-Z])/g, 'sssooo$1sss')
+const restoreUpperCase = str =>
```

Comment:
BTW this implementation will return incorrect results if there's `sss` in the string without the following `ooo`: https://jsbin.com/kawerikaru/edit?js,console,output

Reply:
Yes, an idea here is that no one usually enters our symbols sequence. :smile: 
I've changed that marking-sequence to be `a-a-a-a-a`. Would it be better?